.babelrc.js

```diff
@@ -30,6 +32,11 @@ function istanbulHacks() {
   };
 }
 
+const babylonTransformCharcodes = relative.resolve(
+  "babel-plugin-transform-charcodes",
+  "./packages/babylon"
+);
```

Comment:
Since it's an external package, couldn't we just require it and pass it as a function? Might be easier, right?

Reply:
Since `"babel-plugin-transform-charcodes"` is in a subpackage, we would still need to
```js
relative(
  "babel-plugin-transform-charcodes",
  "./packages/babylon"
);
```

Reply:
Why don't we just add it to the top level?

Reply:
This plugins isn't a subpackage from our repo but a external one. We can add it to the top level one.

Reply:
I initially added it to the top level but, [as pointed out](https://github.com/babel/babel/pull/7240#pullrequestreview-89976862), it would be confusing to see `import "charcodes"` in Babylon without it beeing in Babylon's `package.json`.

Reply:
makes sense, I guess we could put it in both places too